wagtail/contrib/styleguide/templates/wagtailstyleguide/base.html

```diff
@@ -18,7 +18,7 @@
 
     <div class="nice-padding">
         <h2>Contents</h2>
-        <nav role="navigation">
+        <nav>
```

Comment:
I’d like us to keep this to maximise support for those landmarks, unless you have some good understanding that this is not an issue?

Reply:
As per my previous comment, I think we can safely remove this as all the browsers we support are HTML5 compliant, and you have tested in IE11 and Jaws.